### PR TITLE
Merge BRICR Dev into Develop

### PR DESCRIPTION
**Pull request by nllong**
Merge latest changes from BRICR Dev into the development branch.

### CODE REVIEW

**Comment by nllong** [comment]
This is almost a clean merge. @macintoshpie and @adrian-lara can you double-check these conflicts? Thanks!

File: config/settings/travis.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
         'USER': 'postgres',
         'PASSWORD': '',
         'HOST': 'localhost',
-        'PORT': '5432',
+        'PORT': '5433',
```
Comment (nllong) [inline config/settings/travis.py]: I'm guessing that this is here because I cherry picked the change into bricr-dev from develop

**Comment by macintoshpie** [comment]
I just opened another PR here to address the conflict (I don't have push permissions for the bricr-dev branch I believe, so I created another) #2130

**Comment by nllong** [comment]
@adrian-lara this has been superceded by #2130, correct? If so, then please close. Thanks.

**Pull request closed without merging**